priority queue in python

noviicee opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.
You can learn more about contributing to NeoAlgo here: https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md

Happy Contributing!

-->

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added code for priority queue in python and updated the corresponding readme file.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)
closes #6566

noviicee committed:
Merge pull request #1 from TesseractCoding/master

Syncing the fork

noviicee committed:
Added Priority Queue [Python]

noviicee committed:
Update priority_queue.py

noviicee commented:
@HarshCasper kindly review this one.

noviicee commented:
@ankitaggarwal23 can you review this..?
If the pr  is not merged within 2 days..it will be closed along with the issue!

noviicee committed:
Merge branch 'master' into issue-6566

siddharth25pandey commented on Python/ds/priority_queue.py:
```diff
@@ -0,0 +1,118 @@
+    pq1.insert_into_pq('A', 10)
+    pq1.insert_into_pq('Q', 45)
+    pq1.insert_into_pq('D', 4)
+    pq1.insert_into_pq('W', 55)
```
Use Dynamic Input

siddharth25pandey reviewed (changes requested):
@noviicee Do the suggested changes

noviicee committed:
Update priority_queue.py

ankitaggarwal23 reviewed (changes requested):
Few changes: 

- Add Multi-Line Comments 
- Add a beginner-friendly description at the top
- Use proper code Format 
- Remove extra lines of code and spaces
- Add sample I/O at the end of the Code
- Add Time and Space Complexity at the end of the Code   
- Add a new line at the EOF
- Use proper display message how you are reading and writing outputs
- Add input-output prompts to make the program more interactive.

noviicee committed:
Made required changes

I have added multi-line comments wherever necessary, sample test-cases and space-time complexity at the end of the code.
I have also formatted the code according to pep8 guidelines.

noviicee commented:
> Few changes:
> 
> * Add Multi-Line Comments
> * Add a beginner-friendly description at the top
> * Use proper code Format
> * Remove extra lines of code and spaces
> * Add sample I/O at the end of the Code
> * Add Time and Space Complexity at the end of the Code
> * Add a new line at the EOF
> * Use proper display message how you are reading and writing outputs
> * Add input-output prompts to make the program more interactive.

I have added multi-line comments wherever necessary, sample test-cases and space-time complexity at the end of the code.
I have also formatted the code according to pep8 guidelines.
